packages/select/src/components/select/select.tsx

```diff
@@ -39,6 +39,12 @@ import { IQueryListRendererProps, QueryList } from "../query-list/queryList";
 export type SelectProps<T> = ISelectProps<T>;
 /** @deprecated use SelectProps */
 export interface ISelectProps<T> extends IListItemsProps<T> {
+    /**
+     * Whether the component should take up the full width of its container.
+     * This overrides `popoverProps.fill` and `tagInputProps.fill`.
```

Comment:
there is no `tagInputProps` for this component.

```suggestion
     * This overrides `popoverProps.fill`.
```